cmd/disk-cache.go

```diff
@@ -546,7 +552,7 @@ func newCache(config cache.Config) ([]*diskCache, bool, error) {
 		if quota == 0 {
 			quota = config.Quota
 		}
-		cache, err := newDiskCache(ctx, dir, quota, config.After, config.WatermarkLow, config.WatermarkHigh)
+		cache, err := newDiskCache(ctx, dir, quota, config.After, config.WatermarkLow, config.WatermarkHigh, config.Range)
```

Comment:
why isn't `newDiskCache` taking `config` as input directly? rather than individual args? isn't it easier to write and consume?

Reply:
true, fixed this.